lib/utils.pm

```diff
@@ -1154,7 +1154,7 @@ sub ensure_serialdev_permissions {
     # reboot an alternative https://superuser.com/a/609141/327890 would need
     # handling of optional sudo password prompt within the exec
     # Need backwards support for SLES11-SP4 here, the command "gpasswd" and "stat" are only available with SLES-12 at least.
-    if (is_sle && check_var('VERSION', '11-SP4')) {
+    if (is_sle('11-SP4')) {
```

Comment:
Looks like this will die with `Unsupported version parameter`, we should put `=`.